static/main.js

```diff
@@ -6,6 +6,7 @@ const importer = {
       const script = document.createElement('script')
       script.type = 'text/javascript'
       script.src = url
+      script.crossOrigin = 'anonymous'
```

Comment:
Minor point, but the actual name of the property is 'crossorigin' (all lower case).

I know the camel case version works, and I know that html attribute names are case-insensitive, but I think I'd prefer to use the casing indicated by the spec.

Reply:
According to MDN, the proper name is `crossOrigin` when we refer to it from JS https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement
The fix won't work if the casing is changed to all lower case.